lib/galaxy/datatypes/phylip.py

```diff
@@ -59,12 +59,22 @@ def sniff_prefix(self, file_prefix):
         f = file_prefix.string_io()
         # Get number of sequence from first line
         nb_seq = int(f.readline().split()[0])
```

Comment:
```suggestion
        # Get number of sequences and sequence length from first line
        nb_seq, _seq_length = (int(n) for n in f.readline().split())
```

Reply:
Isn't that a linting error ?
```suggestion
        # Get number of sequences and sequence length from first line
        nb_seq, _ = (int(n) for n in f.readline().split())
```
?

Reply:
No, because I've used a variable name (`_seq_length`) starting with an underscore.